### PR TITLE
TST: Add failing test for GH 1423

### PR DESCRIPTION
**Pull request by sandhujasmine**
Add failing test and mark with xfail for [GH 1423](https://github.com/blaze/blaze/issues/1423)

To run the test using docker provisioning.sh script, after odo [PR#428](https://github.com/blaze/odo/pull/428) is merged; run the ./provisioning.sh and add the blaze/examples/data as a mount point for docker so it can load data into postgres.

Example:
`provisioning.sh -t ~/tmp/docker/ -m ${BLAZE_PROJ_PATH}/blaze/blaze/examples/data restart`

### CODE REVIEW

File: blaze/compute/tests/test_postgresql_compute.py [hunk 1]
```diff
@@ -575,11 +575,21 @@ def test_interactive_len(sql):
...
-def test_sample(sql):
-    t = symbol('t', discover(sql))
-    result = compute(t.sample(n=1), sql)
+def test_sample_n(nyc):
+    t = symbol('t', discover(nyc))
+    result = compute(t.sample(n=14), nyc)
     s = odo(result, pd.DataFrame)
-    assert len(s) == 1
-    result2 = compute(t.sample(frac=0.5), sql)
+    assert len(s) == 14
```
Comment (llllllllll) [inline blaze/compute/tests/test_postgresql_compute.py]: should we check that the sorted results are the same ?

**Comment by llllllllll** [comment]
Thanks for adding an xfail test. This is great way to codify tracking this issue!

File: blaze/compute/tests/test_postgresql_compute.py (hunk 1 above)
Comment (sandhujasmine) [inline blaze/compute/tests/test_postgresql_compute.py]: Not sure I understand - could you explain a bit more please.

The `sample()` method will by default do a random sampling of the data in the SQL query by invoking `ORDER BY random()`.

In both cases we're doing a random sampling; however when using  `n=100` we have LIMIT defined as 100 while for `frac=0.5`, the LIMIT is computed as `COUNT(*) * 0.5`

Comment (llllllllll) [inline blaze/compute/tests/test_postgresql_compute.py]: sorry, maybe I incorrectly assumed that we were doing a sample of the whole table which should just shuffle it. If this is a subset then I think that we should test that all the records exist in the base table as a sanity check that we are actually sampling correcectly.

**Comment by kwmsmith** [comment]
Closing as this is subsumed in #1452.

**Pull request closed without merging**